src/Perlang.Tests.Integration/EvalHelper.cs

```diff
@@ -303,32 +303,34 @@ internal static object EvalWithArguments(string source, params string[] argument
 
         private static void AssertFailScanErrorHandler(ScanError scanError)
         {
-            throw new Exception("ScanError occurred. See inner exception for details.", scanError);
+            // Note: does not contain stack trace, so no point in wrapping in Exception() at this point.
+            throw scanError;
         }
 
         private static void AssertFailParseErrorHandler(ParseError parseError)
         {
-            throw new Exception("ParseError occurred. See inner exception for details.", parseError);
+            // Note: does not contain stack trace, so no point in wrapping in Exception() at this point.
+            throw parseError;
```

Comment:
These hunks revert part of #246; it doesn't add any value to wrap these exceptions, since they haven't been thrown yet in the first place => no stack trace has been added to the `ScanError`/`ParseError` objects. We might as well just keep it like it used to be for now.